web/src/components/admin/settings/queue/AdminQueueStats.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div v-if="stats" class="flex justify-center">
+    <div class="bg-gray-100 dark:bg-dark-gray-600 text-color dark:text-gray-400 rounded-md py-5 px-5 w-full">
+      <div class="flex w-full">
+        <h3 class="text-lg font-semibold leading-tight uppercase flex-1">
+          {{ $t('admin.settings.queue.stats.completed_count') }}
+        </h3>
+      </div>
+      <div class="relative overflow-hidden transition-all duration-500">
+        <div>
+          <div class="pb-4 lg:pb-6">
+            <h4 class="text-2xl lg:text-3xl font-semibold leading-tight inline-block">
+              {{ stats.completed_count }}
```

Comment:
Unluckily this is not set in the api endpoint yet 🙈 But really cool to know this info at some point.